datadog/client_test.go

```diff
@@ -125,3 +168,29 @@ func BenchmarkClient(b *testing.B) {
 		})
 	}
 }
+
+// startUDPListener starts a go routine listening for UDP packets on 127.0.0.1 and an available port.
+// The address listened to is returned as `addr`. The payloads of packets received are copied to `packets`
+func startUDPListener(t *testing.T, packets chan []byte) (addr string, closer io.Closer) {
```

Comment:
add `t.Helper()` here?

Reply:
I'm not familiar with `t.Helper()` In the docs it says:
"Helper marks the calling function as a test helper function. When printing file and line information, that function will be skipped."

I don't see any other documented effects of calling t.Helper(). Not printing the full stack trace on errors is not behavior I would want. Am I missing something?

Reply:
You don't get a full stacktrace on t.Fatal anyway, just the failing line. With t.Helper It'll print the line from the test that called it instead of the line from the helper, which I find is useful.

You should be able to verify this by manually making the test fail and then running `go test ./...`

Reply:
Oh, I see how that's useful now. Already merged the PR, but I know why you'd use t.Helper for the future now. Thanks!